### DISCUSSION TITLE
Welcome to go2cs Discussions!

### DISCUSSION THREAD

**Discussion by ritchiecarroll** [ritchiecarroll/go2cs#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PPC-FGao** [comment]
I am new for go2cs. I followed readme instructions on "https://github.com/GridProtectionAlliance/go2cs?tab=readme-ov-file". I tried to convert a simple "Hello World" Go example but always failed with message "Error: Unable to resolve GOROOT src directory: "C:\Go\src\". Validate that Go is properly installed." Here are steps I took:
1. download and install Go by following instruction on https://go.dev/doc/install.
2. verified it's installed on my local folder "C:\Program Files\Go". (I assume it is the %GOROOT%).
3. check Environment Variables on local system. There is GOPATH with value of C:Users\username\go. There is no such GOROOT in list.
4. followed deployment steps on "https://github.com/GridProtectionAlliance/go2cs/releases".
4.1. Download dependencies.zip and extract to %GOPATH%\src (i.e. C:Users\username\go\src)
4.2. Download the [go2cs-win-x64.zip] and copy to %GOPATH%\bin (i.e. C:Users\username\go\bin)
5. followed steps on "https://go.dev/doc/tutorial/getting-started". write, run, and save the code on C:\Users\username\source\go-repos\hello
6. In local powerShell, under C:Users\username\go\bin, run "go2cs -s -r C:\Users\username\source\go-repos\hello"
7. After shows "Starting Go code pre-scan to update metadata...", It shows "Error: Unable to resolve GOROOT src directory: "C:\Go\src\". 
Validate that Go is properly installed."

Which step I missed or did wrong?

Thanks
